scrapy/extensions/feedexport.py

```diff
@@ -86,7 +86,9 @@ class IFeedStorage(Interface):
 
     def __init__(uri, *, feed_options=None):
         """Initialize the storage with the parameters given in the URI and the
-        feed-specific options (see :setting:`FEEDS`)"""
+        feed-specific options (see :setting:`FEEDS`).
+        CAREFUL: in `feed_options` the default `overwrite=True` will cause you to 
+        lose the previous version of your data"""
```

Comment:
This would need some format adjustments to adhere to reStructuredText syntax, but more importantly this is not accurate here (overwrite is not True for IFeedStorage by default ), and this is not where we need to cover this, since this docstring is not currently rendered into our documentation.

We need to update `docs/topics/feed-exports.rst` instead, both the section about the `FEEDS` option, to mention the default value for GCS, and the sections of the specific storage backends that use overwrite by default, to add a warning like the one you are adding here.